frameworks/elastic/tests/test_tls.py

```diff
@@ -116,5 +117,12 @@ def test_tls_recovery(elastic_service: Dict[str, Any], service_account: Dict[str
 
     for pod in json.loads(stdout):
         sdk_recovery.check_permanent_recovery(
-            package_name, service_name, pod, recovery_timeout_s=60 * 60
+            package_name, service_name, pod, recovery_timeout_s=25 * 60
+        )
+        # Remove before merge
```

Comment:
Why should it be removed? We should wait for a GREEN health status after a pod replace, no?

Reply:
I suppose, after adding 3rd data node it should work without waiting. I add this just to check in log that state is ok.

Reply:
We should wait in any case, because starting another pod replace when there might be still unassigned shards will potentially cause data loss, and manifest as flakes.

Reply:
make sense. thanks!